### DISCUSSION TITLE
May I have some quick documentation on DASM lists and symbol maps

### DISCUSSION THREAD

**Discussion by Pokeman2003** [stella-emu/stella#1060] [Q&A] [answered]
DASM doesn't actually have documentation for either of these file formats. I've managed to produce my own after struggling with a Windows kernel call malfunctioning, but now I'm not entirely sure what I'm looking at. I know these files are pretty simple, but there are couple things left unexplained, such as the (R ) symbol occasionally seen in my symbol map, and I'm worried I'm missing something in the list file that didn't appear from my test.

If you're curious I'm asking here: The purpose is so I can write a converter to convert WLA-DX symbol maps and list files into DASM ones so I can better debug my code on Stella.

**Comment by thrust26** [comment]
I am also not very familiar with these files either. Have you tried asking e.g. in the DASM club at AtariAge?

**Reply by Pokeman2003** [reply]
I have not yet, but that's because I don't remember my AtariAge password, and I'm locked out of the email I used to sign up. (I plan to recover it later today but I don't have the time right this second.

**Reply by thrust26** [reply]
Try contacting Albert directly by mail. 

albert@atariage.com

**Comment by andrew-davie** [comment, marked as answer]
Actually, (R  ) is mentioned in the DASM manual.
It tells you if a symbol has not only been *defined*, but also Referenced.

**Reply by Pokeman2003** [reply]
For some reason I completely and utterly failed to notice that. Thank you.
